imap_processing/glows/l0/glows_l0_data.py

```diff
@@ -152,50 +132,61 @@ class DirectEventL0(GlowsL0):
     MET : int
         CCSDS Packet Time Stamp (coarse time)
     SEC : int
-        Data timestamp, seconds counter.
+        Data IMAP timestamp, seconds counter.
     LEN : int
         Number of packets in data set.
     SEQ : int
         Packet sequence in data set.
+    DE_DATA : bytearray
+        Raw direct event data (compressed)
     ground_sw_version : str
         Ground software version
     packet_file_name : str
         File name of the source packet
     ccsds_header : CcsdsData
         CCSDS header data
+
+    Methods
+    -------
+    sequence_match_check
+        Compare two DirectEventL0 objects and see if MET and LEN are equal
+
     """
 
     MET: int
     SEC: int
     LEN: int
     SEQ: int
+    DE_DATA: bytearray
+
+    def __post_init__(self):
+        """Convert from string to bytearray if DE_DATA is a string of ones and zeros."""
+        if isinstance(self.DE_DATA, str):
+            # Convert string output from space_packet_parser to bytearray
+            self.DE_DATA = bytearray(
+                int(self.DE_DATA, 2).to_bytes(len(self.DE_DATA) // 8, "big")
```

Comment:
Is the byte order part something that GLOWS told you?

Reply:
Also, a few more dumb questions about this:

1. When you convert the DE_DATA string to an int, would it cause problems if the first character in the string is a 0 since it'll drop the 0 when it converts to an int?
2. Is the bytearray basically the binary string, grouped into 1 byte segments?